build-package.sh

```diff
@@ -351,11 +351,11 @@ termux_step_start_build() {
 
 	local TERMUX_ELF_CLEANER_SRC=$TERMUX_COMMON_CACHEDIR/termux-elf-cleaner.cpp
 	local TERMUX_ELF_CLEANER_VERSION=$(bash -c ". $TERMUX_SCRIPTDIR/packages/termux-elf-cleaner/build.sh; echo \$TERMUX_PKG_VERSION")
-	termux_download \
-		https://raw.githubusercontent.com/termux/termux-elf-cleaner/v$TERMUX_ELF_CLEANER_VERSION/termux-elf-cleaner.cpp \
-		$TERMUX_ELF_CLEANER_SRC \
-		11a38372f4d0e36b7556382c7ecffecae35cee8b68daaee2dbee025f758e17ee
-	if [ "$TERMUX_ELF_CLEANER_SRC" -nt "$TERMUX_ELF_CLEANER" ]; then
+	if [ ! -f $TERMUX_ELF_CLEANER ]; then
```

Comment:
The checksum given, `11a38372f4d0e36b7556382c7ecffecae35cee8b68daaee2dbee025f758e17ee`, effectively does that - if there is an existing file already matching the checksum it is kept without downloading it again.